module/applications/global/prompt-factory.mjs

```diff
@@ -433,7 +479,20 @@ class ItemPromptFactory extends PromptFactory {
   }
 
   async _talentCategoryPrompt() {
-    const buttons = Object.entries( ED4E.talentCategory ).map(
+
+    const versatilityId = game.settings.get( "ed4e", "edidVersatility" );
```

Comment:
Small thing: maybe call it `versatilityEdid` to be even more descriptive. Might be confused with the id of an actual item or something...